### ISSUE TITLE
Implementations

### ISSUE THREAD

**Issue by Davides98** [messerli90/laravel-vue-kanban-tutorial#2]
Can you implement the creation of variable columns?
in the sense that the user can decide the name and quantity to be assigned to each single column.

It would also be convenient to create multiple projects
Having this hierarchy
Project 1 -> Colums-> task
Project 2 -> Colums-> task

**Comment by messerli90** [comment]
Definitely. I've been busy with some other projects this last month. But want to get to Part 2 soon, and I'll talk about adding/editing columns.

**Comment by Davides98** [comment]
Hi, I hope it's soon because it's a really nice project.

However, analyzing it would be very convenient to create the tasks through a modal where all the various parameters are set, and perhaps the subtasks (for example checklist).

If you want I can list the whole thing better.

I wanted to point out that this is one of the best tutorials on the web and the description in the blog is very clear.

**Comment by Davides98** [comment]
Have you got news for part 2?

**Comment by DeGroote99** [comment]
Any news?
